data/json/construction.json

```diff
@@ -1537,6 +1537,17 @@
     "pre_special": "check_empty",
     "post_terrain": "f_straw_bed"
   },
+  {
+    "type": "construction",
+    "description": "Build Pile of Leaves Bed",
+    "category": "FURN",
+    "required_skills": [ [ "survival", 0 ] ],
+    "time": "1 m",
+    "components": [ [ [ "withered", 10 ] ] ],
```

Comment:
a fiber mat requires 12 + plant fiber, and is still too thin to be used as a sleeping place.  This should be about 60 or maybe 100.

Reply:
The description of the fiber mat compares it to a picnic blanket, so I imagine it is wider and longer than a place to sleep.

I made a test about collecting withered plants. Going around in the middle of the forest, smashing underbrush with a heavy stick, putting the plants in a duffel bag, it took me an hour and ten minutes to collect 60 of them, and two hours and two minutes to collect 100 of them. This job would be around 15 minutes at most. The whole thing is highly abstracted, but I don't think it would take more than 50 plants.
Determining the material requirements I used the 8 piles of straw cost of the straw bed, and the 40 piles of straw and 40 withered plants cost of the hay bale.